metpy/plots/skewt.py

```diff
@@ -830,18 +829,28 @@ def plot_colormapped(self, u, v, c, bounds=None, colors=None, **kwargs):
         --------
         :meth:`Hodograph.plot`
 
+        Notes
+        -----
+        `plot_colormapped` previously accepted `bounds` as a parameter for coordinate values.
+        This has been deprecated in 0.11 in favor of `intervals`.
+
         """
+        if bounds is not None:
```

Comment:
And add before here:
```python
bounds = kwargs.get('bounds')
```
That allows us to still support code that does `plot_colormapped(u, v, heights, bounds=..., colors=...)` and code that does `plot_colormapped(u, v, heights, mybounds, mycolors)`.